irc3/plugins/command.py

```diff
@@ -268,7 +268,7 @@ def split_command(self, data, use_shlex=True):
             if use_shlex:
                 return shlex.split(data)
             else:
-                return data.split()
+                return data.split(' ')
```

Comment:
I'm not completely sure, but I think only the server is affected by this, since client commands have `use_shlex` set to `True`. Also, the server isn't supposed to alter the messages at all.